content/broker/broker-transition/usage/rest/_index.nb.md

```diff
@@ -0,0 +1,349 @@
+            "ParentReceiptID": null,
+            "LastChanged": "2024-05-08T12:33:14.819",
+            "Status": "Rejected",
+            "Text": "File failed during upload processing in Altinn 3.",
+            "SendersReference": null,
+            "ServiceOwnerPartyReference": null,
+            "PartyReference": "313559017",
+            "ReceiptHistory": null,
+            "SubReceipts": null
+        },
+        {
+            "ReceiptID": 0,
+            "ParentReceiptID": null,
+            "LastChanged": "2024-05-08T12:33:14.819",
+            "Status": "Rejected",
+            "Text": "File failed during upload processing in Altinn 3.",
+            "SendersReference": null,
+            "ServiceOwnerPartyReference": null,
+            "PartyReference": "314126866",
+            "ReceiptHistory": null,
+            "SubReceipts": null
+        }
+    ]
+}
+```
+
+#### Differences
+
+For Altinn 3 Overgangs-tjenester er kvitteringen en pseudo-kvittering bygget av Altinn 3 Broker fil metadata.
+
+## Inbox (recipient)
+
+### Bekreft om brukere har filer tilgjengelig
+Denne operasjonen verifiserer om noen av de gitte mottakerne har noen filer tilgjengelig for nedlasting for den gitte ServiceCode / ServiceEdition kombinasjonen.
+
+Hvis filer er tilgjengelige, returnerer det sant, ellers returnerer det usant.
+
+Implementeringen skiller seg ikke ut for Altinn 3 overførte tjenester.
+
+Header
+```HTTP
+GET https://www.altinn.no/api/brokerservice/inbox/hasavailablefiles?serviceCode={serviceCode}&serviceEditionCode={serviceEdition}&recipients={recipient1},{recipient2} HTTP/1.1 
+accept: application/json
+ApiKey: myKey
+```
+
+### Få mottakers inbox
+Denne operasjonen henter en liste over filer tilgjengelig for brukeren.
+
+Denne operasjonen vil bare returnere fil-listen for overførte tjenester hvis tjenesten er spesifisert i forespørselen.
+
+Header
+```HTTP
+GET https://www.altinn.no/api/{who}/brokerservice/inbox/?serviceCode={serviceCode}&serviceEditionCode={serviceEdition} HTTP/1.1
+accept: application/json
+ApiKey: myKey
+Content-Type: application/zip
+```
+
+#### Eksempel kvittering
+```JSON
+{
+    "ReceiptID": 0,
+    "ParentReceiptID": null,
+    "LastChanged": "2024-05-08T12:14:52.288",
+    "Status": "Ok",
+    "Text": "A file has been made available for download.",
+    "SendersReference": null,
+    "ServiceOwnerPartyReference": null,
+    "PartyReference": "313559017",
+    "ReceiptHistory": null,
+    "SubReceipts": null
+}
+```
+
+#### Forskjeller
+Inbox operasjonen vil som standard hente en liste over Altinn 2 filer.Hvis det ikke leveres noen tjenestekode/tjenesteutgavekode, vil operasjonen bare liste opp filer fra Altinn 2.
+Kvitteringen vil ikke ha en ReceiptID, ParentReceiptID, SendersReference eller ServiceOwnerPartyReference.
+
+### GetFileDetails inbox (mottaker)
+Denne operasjonen skiller seg ikke fra GetFileDetails for [outbox(sender)](#get-file-details-outbox-sender), bortsett fra i URI for forespørselen.
```

Comment:
I've tested them and they seem to work, linking back to the correct heading in the same language site. But I think that's mainly because I haven't renamed the headings since they are named after operations.